docs/install.md

```diff
@@ -1,77 +1,220 @@
+    *   Go to Datastore in the cloud console
+    *   Hit the `Create Entity` button
+    *   Select the region that your data should be in. No need to create any
+        Entities after selecting your region
+*   Enable [Cloud Deployment Manager
+    V2](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
+    for managing the GCP via the CLI tool `gclou`
+
+## Local Turbinia
+
+*   If Turbinia will run on local machines, jump to [Core Installation
+    Steps](#core-installation-steps)
+*   Otherwise, follow [GCP Turbinia](#gcp-turbinia)
+
+## GCP Turbinia
+
+The following is a one possible configuration and setup for Turbinia in GCP.
+This is still a rough process and future versions will be containerized.
+
+**NOTE:** GCP Turbinia requires two Google Cloud Engine (GCE) instance types:
+1 server and 1 or more workers
+
+### 1. Create a GCE Instance as Server
+*   Create a [new GCE
+    instance](https://console.cloud.google.com/compute/instances) from a recent version
+    of Debian or Ubuntu
+    *   This should work on other Linux flavors, but these are untested. Feel
+        free to test and fix them ;)
+*   Follow [Core Installation Steps](#core-installation-steps)
+
+### 2. Create a Google Cloud Storage (GCS) Bucket
+
+**NOTE:** GCS FUSE is used here for convenience to keep scripts and log files,
+but this isn't strictly necessary for Turbinia to run if you have an alternate
+means to save logging data.
+
+*   [Create a new GCS
+    bucket](https://cloud.google.com/storage/docs/creating-buckets) and create
+    directories for `scripts` and `output/logs`
+*   Mount your GCS bucket on your server instance using [GCS
+    FUSE](https://cloud.google.com/storage/docs/gcs-fuse)
+*   cp `turbinia/tools/gcp_init/*.sh` into your locally mounted copy of
+    `$your_bucket/scripts`
+*   Edit the variables in `scripts/start-wrapper.sh` and
+    `scripts/start-turbinia-common.sh` as appropriate
+*   ***NOTE:*** The `start-wrapper.sh` script has a `GOOGLE_APPLICATION_CREDENTIALS`
+    environment var in the middle of the script that needs to be updated
+
+### 3. Create an Instance Template (Prep Work for Worker)
+*   Stop the server instance
+*   Create a new image from the server VM's disk
+*   Create a new Instance Template using the newly created image
+
+### 4. Create a GCE Instance as Worker
+*   Create a new Managed Instance Group from the newly created Instance Template
+*   In your worker VM, add a new custom metadata key `startup-script-url`
+    pointing to `gs://$your_bucket/scripts/start-wrapper.sh`
+*   TODO(beamcodeup): Update this section when Issue #132 is resolved.
+*   Upon start, your VM should mount your GCS Bucket, and copy the start scripts
+    into the home directory of the Turbinia user and will then start the
+    Turbinia worker
+*   ***NOTE:*** If the GCS FUSE is used for log files, update the Turbinia config
+    file with a new `LOG_FILE` path to avoid writing to the server logs.
+
```

Comment:
Not sure what is meant by "to avoid writing to the server logs" here.   Maybe you mean something like "to write server logs directly into GCS"?

Reply:
done